opensrp-chw/src/main/java/org/smartregister/chw/presenter/AboveFiveChildProfilePresenter.java

```diff
@@ -0,0 +1,35 @@
+package org.smartregister.chw.presenter;
+
+import android.app.Activity;
+
+import org.smartregister.chw.activity.AboveFiveChildProfileActivity;
+import org.smartregister.chw.core.contract.CoreChildProfileContract;
+import org.smartregister.chw.interactor.ChildProfileInteractor;
+import org.smartregister.chw.model.ReferralTypeModel;
+import org.smartregister.chw.util.Utils;
+
+import java.lang.ref.WeakReference;
+import java.util.List;
+
+public class AboveFiveChildProfilePresenter extends ChildProfilePresenter {
+
+    private List<ReferralTypeModel> referralTypeModels;
```

Comment:
Codacy found an issue: [Perhaps 'referralTypeModels' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5868622)